### ISSUE TITLE
Issue with "HTMLSelectElement.setCustomValidity()": (short summary here please)

### ISSUE THREAD

**Issue by tpajor** [mdn/content#6478]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/setCustomValidity

#### What information was incorrect, unhelpful, or incomplete?
this feature is not supported by Chrome

#### Specific section or headline?
Browser support

#### What did you expect to see?
info, that this is not supported by Chrome

#### Did you test this? If so, how?
I set custom validity and it works on Firefox, but Chrome still uses it's default tooltip text

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/htmlselectelement/setcustomvalidity`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/setCustomValidity
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/htmlselectelement/setcustomvalidity/index.html
* Last commit: https://github.com/mdn/content/commit/3a5d65124798f70ef4777aef7ab11b7e9a5b7dbf
* Document last modified: 2021-06-13T14:10:35.000Z

</details>

**Comment by sideshowbarker** [comment]
The support information is correct; Chrome is known to have had support for setCustomValidity() for a very long time now.

If you have new information which indicates otherwise, please post reproducible code that demonstrates the problem, and then this could be revisited.

**Comment by tpajor** [comment]
`<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        .c-input {
            width: 26.5em;
            height: 3em;
        }
    </style>
</head>
<body>
<div>
    <input id="login-input" class="c-input" type="email" value="" placeholder="Email" required/>
</div>
<script>
  var loginInput = document.getElementById('login-input');
  loginInput.addEventListener('focus', function (event) {
    event.preventDefault();
    loginInput.setCustomValidity('');
  }, false);
  loginInput.addEventListener('blur', function (event) {
    event.preventDefault();
    if (!loginInput.checkValidity()) {
      loginInput.classList.add('c-input--invalid');
      loginInput.setCustomValidity('Please enter your full email address');
      loginInput.reportValidity();
    }
  }, false);
</script>
</body>
</html>`

Here you go. Does not work on chrome, tested on firefox and it works fine.

**Comment by tpajor** [comment]
```<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <style>
        .c-input {
            width: 26.5em;
            height: 3em;
        }
    </style>
</head>
<body>
<div>
    <input id="login-input" class="c-input" type="email" value="" placeholder="Email" required/>
</div>
<script>
  var loginInput = document.getElementById('login-input');
  loginInput.addEventListener('focus', function (event) {
    event.preventDefault();
    loginInput.setCustomValidity('');
  }, false);
  loginInput.addEventListener('blur', function (event) {
    event.preventDefault();
    if (!loginInput.checkValidity()) {
      loginInput.classList.add('c-input--invalid');
      loginInput.setCustomValidity('Please enter your full email address');
      loginInput.reportValidity();
    }
  }, false);
</script>
</body>
</html>
Chrome ignores custom validity, tested on firefox for cross reference and it works fine.

**Comment by teoli2003** [comment]
Technically, there is no `<select>` element in this example, but `<input>`. So, it is `HTMLInputElement.setCustomValidity()` that is used. This shouldn't change much as Chrome should also [support it](https://github.com/mdn/browser-compat-data/blob/main/api/HTMLInputElement.json#L2056). (We just don't have `HTMLInputElement.setCustomValidity()` documented on MDN).

Then, I tried this simplified test on Firefox and Chrome. Here is the screenshot on Chrome, where the custom validity is indeed correctly shown:

<img width="1671" alt="Screenshot 2021-07-01 at 10 38 16" src="https://user-images.githubusercontent.com/1466293/124094032-ae3ad300-da58-11eb-8604-c602f622310a.png">

I suspect that something else is going in the way, but not `setCustomValidity()`.

**Comment by tpajor** [comment]
Maybe they did not implement custom validity on email input, because they went for more detailed native validation there? This is probably a bug on Chrome's side.

**Comment by teoli2003** [comment]
Maybe it is linked to this: https://bugs.chromium.org/p/chromium/issues/detail?id=828757

**Comment by sideshowbarker** [comment]
> Maybe it is linked to this: [bugs.chromium.org/p/chromium/issues/detail?id=828757](https://bugs.chromium.org/p/chromium/issues/detail?id=828757)

Yeah, that sounds like the same problem that @tpajor is describing; in particular, at https://bugs.chromium.org/p/chromium/issues/detail?id=828757#c4, I see:

> It seems this is a real bug. If we hover on the `<input>` without typing anything, it shows "Please fill out this field" tooltip. However, after calling setCustomValidity(), no tooltip is shown.

**Comment by sideshowbarker** [comment]
This is, anyway, related to browser-support info — which is an issue for BCD, not MDN itself. So, moving this to the BCD repo.
